tests/pkg_cryptoauthlib_internal-tests/Makefile

```diff
@@ -2,6 +2,7 @@ BOARD ?= nucleo-f767zi
 
 include ../Makefile.tests_common
 
+CFLAGS += -DTHREAD_STACKSIZE_MAIN=\(4*THREAD_STACKSIZE_DEFAULT\)
```

Comment:
Just out of curiosity, how much stack does the test require, have you checked that?

Reply:
2652 Bytes. I'll  reduce it to 3*THREAD_STACKSIZE_DEFAULT.